### ISSUE TITLE
Python 3.6.6, ModuleNotFoundError: import of os halted; None in sys.modules

### ISSUE THREAD

**Issue by vkatochka** [pycontribs/tendo#33]
Hi,

I found an example "single instance of program" on https://stackoverflow.com/a/1265445
I ran the script and got an error.

> python3 test.py
~~~
#
import sys
from tendo import singleton

try:
    me = singleton.SingleInstance()
except:
    sys.exit()
~~~

Exception ignored in: <bound method SingleInstance.__del__ of <tendo.singleton.SingleInstance object at 0x7f4cb68f3f28>>
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/tendo/singleton.py", line 70, in __del__
ModuleNotFoundError: import of os halted; None in sys.modules

Ubuntu 18.04.1 LTS, Python 3.6.6

**Comment by vkatochka** [comment]
I installed latest version like this:
> pip3 install git+git://github.com/pycontribs/tendo.git@master
> pip3 show tendo

Name: tendo
Version: 0.2.13.dev7
Summary: A Python library that extends some core functionality
Home-page: https://github.com/pycontribs/tendo
Author: Sorin Sbarnea
Author-email: sorin.sbarnea@gmail.com
License: BSD
Location: /usr/local/lib/python3.6/dist-packages
Requires: pbr, pip, setuptools, six

But I am still has error:
~~~
Exception ignored in: <bound method SingleInstance.__del__ of <tendo.singleton.SingleInstance object at 0x7f74cd250630>>
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/tendo/singleton.py", line 72, in __del__
ModuleNotFoundError: import of os halted; None in sys.modules
~~~

**Comment by fx-kirin** [comment]
This will be fixed by #38.
